### ISSUE TITLE
Maximum of 24 items shown in Choose Instrument List

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#26866]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

It appears there is a maximum number of 24 search results shown in the results list in Choose Instruments in the New Score Wizard. This can result in missing instruments, particularly when the category is set to All Instruments.  This is likely to be an issue with arrow keys and keyboard navigation if other users haven't encountered it and may be related to a similar issue with max of 20 instruments in the export parts dialogue box.

Steps:
1. From the File menu select New, or press Ctrl+N.
The New Score wizard appears.
2. Select the Choose Instruments radio button.
3. Select the All Instruments option from the combo box.
Expected behaviour is that all instruments are now available.
4. (example test) Select Woodwinds category and review search results.
Actual behaviour: Oboe, Bassoon are not listed.
5. (example test) Select Brass category and review the search results.
Actual behaviour: Trombone is not listed.
6. (example test) Select Strings plucked category and review the search results.
Actual behaviour: Bass guitar is not listed.
It appears the search results are limited to 24 results, meaning many instruments are not listed.
7. Note that if you type a letter in the search box, a different 24 results are shown, based on all categories, not the selected one.

### Supporting files, videos and screenshots

N/A

### What is the latest version of MuseScore Studio where this issue is present?

4.4

### Regression

Yes, this used to work in MuseScore 3.x and now is broken

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
